src/coreclr/src/binder/bindertracing.cpp

```diff
@@ -231,3 +259,105 @@ namespace BinderTracing
         return m_ignoreBind;
     }
 }
+
+namespace BinderTracing
+{
+#ifdef FEATURE_EVENT_TRACE
+    void ResolutionAttemptedOperation::FireEventForStage(Stage stage)
+    {
+        if (stage == Stage::NotYetStarted)
+        {
+            return;
+        }
+
+        StackSString errorMsg, resultAssemblyName, resultAssemblyPath;
+        uint16_t result;
```

Comment:
Nit: Declare this as the enum type so that instead of casting on each assignment, you can cast only once at the end when using it.